src/redux/actions.js

```diff
@@ -0,0 +1,6 @@
+export function updateUser (user) {
```

Comment:
I would make all the actions constants for good practice and access those constants in reducer

Reply:
How do we do that?